.github/workflows/push_nightly_docker_ghcr.yml

```
@@ -0,0 +1,19 @@
+name: Build PyTorch nightly Docker image and push to GitHub Container Registry
+on:
+  schedule:
+    # Push the nightly docker daily at 1 PM UTC
+    - cron: '0 13 * * *'
+  # Have the ability to trigger this job manually using the API as well
+  workflow_dispatch:
+
+jobs:
+  build-publish-docker:
+    runs-on: ubuntu-18.04
```

```suggestion
    if: ${{ github.repository_owner == 'pytorch' }}
    runs-on: ubuntu-18.04
```

We should add this so that this doesn't accidentally get triggered on forked repositories and fail out there.

Similar to https://github.com/pytorch/pytorch/blob/d90911adf97c0e3dc7829e232eff291932210a6f/.github/workflows/update_s3_htmls.yml#L12

---

Makes sense. Thanks!